src/platforms/react-native/manual-setup/hermes.mdx

```diff
@@ -24,30 +24,38 @@ If you are using Hermes, you should not have the [RAM bundles](/platforms/react-
 
 ## Source Maps
 
-If you do not need custom sourcemaps, the `sentry.gradle` build step fully supports Hermes source maps.
+If you do not need custom source maps, the `sentry.gradle` build step fully supports Hermes source maps.
 
 ## Custom Source Maps
 
-If you are manually bundling and building React Native apps, follow this three-step process.
+To manually generate source maps follow this three-step process. Be aware that it's important to exactly match the steps of your React Native tooling. Otherwise the generated bundle and source maps might be working incorrectly, for example pointing to unrelated lines. The following command are examples of the default flow.
 
-### Compile Sourcemaps
+The React Native tooling steps are extracted from [`react-native-xcode.sh`](https://github.com/facebook/react-native/blob/0.71-stable/scripts/react-native-xcode.sh) for iOS and from [`BundleHermesCTask.kt`](https://github.com/facebook/react-native/blob/0.71-stable/packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/BundleHermesCTask.kt) for Android. 
 
 <Note>
 
-Currently, there is an issue when bundling the `hermes` bundle and `source maps` manually.
-We recommend not bundling these manually until this is fixed.
-Follow the [issue](https://github.com/facebook/react-native/issues/34043) for more details.
+[`BundleHermesCTask.kt`](https://github.com/facebook/react-native/blob/0.71-stable/packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/BundleHermesCTask.kt) is only available from `0.71`. Version before use [`react.gradle`](https://github.com/facebook/react-native/blob/0.70-stable/react.gradle).
 
 </Note>
 
+### Compile Source maps
+
+Depending on how your app bundle was generated not all the steps are required. If you use `EXTRA_PACKAGER_ARGS` on `iOS` only the first step applies. This is the default behavior if you used [`@sentry/wizard`](/platforms/react-native/#install). If you use [`SOURCEMAP_FILE`](https://reactnative.dev/docs/sourcemaps#enable-source-maps-on-ios) all steps are necessary.
+
```

Comment:
```suggestion
- If you use Hermes with JS Bundle only the first step applies, but you must replace `main.jsbundle.packager.map` and `index.android.bundle.packager.map` with `main.jsbundle.map` and `index.android.bundle.map`, respectively, in the following commands.

```

Reply:
Related comment for removing the Note below

Reply:
This also looked confusing to me because `main.jsbundle.packager.map` doesn't appear in the below commands (but `main.jsbundle.map` does) and the reverse is also true, `index.android.bundle.packager.map` DOES appear in the below commands (but `index.android.bundle.map` doesn't). 

So I'm wondering if something is backwards here?